examples/active_replication/source.py

```diff
@@ -0,0 +1,26 @@
+import time
+from erdos.data_stream import DataStream
+from erdos.message import Message
+from erdos.op import Op
+from erdos.timestamp import Timestamp
+from erdos.utils import setup_logging
+
+
+class Source(Op):
+    def __init__(self, name, log_file_name=None):
+        super(Source, self).__init__(name)
+        self._logger = setup_logging(self.name, log_file_name)
+        self._cnt = 0
+        self._seq_num = 0
+
+    @staticmethod
+    def setup_streams(input_streams):
+        return [DataStream(name='input_stream')]
+
+    def execute(self):
+        while self._seq_num < 50:
```

Comment:
Please use self._num_messages instead of 50, and pass num_messages in the constructor with a default value of 50.